generator/template/process.go

```diff
@@ -61,6 +63,52 @@ func processSQLBuilder(dirPath string, dialect jet.Dialect, schemaMetaData metad
 	processTableSQLBuilder("table", sqlBuilderPath, dialect, schemaMetaData, schemaMetaData.TablesMetaData, sqlBuilderTemplate)
 	processTableSQLBuilder("view", sqlBuilderPath, dialect, schemaMetaData, schemaMetaData.ViewsMetaData, sqlBuilderTemplate)
 	processEnumSQLBuilder(sqlBuilderPath, dialect, schemaMetaData.EnumsMetaData, sqlBuilderTemplate)
+	processTableSQLBuilderSetSchema(sqlBuilderPath, schemaMetaData, sqlBuilderTemplate)
```

Comment:
This method will be also useful for views as well. Not just tables.

Reply:
I don't think we need another method, we can reuse `processTableSQLBuilder`, it used for both tables and views.

Reply:
Great idea. I'm thinking of a `generateSetSchema` method that can be called from `processTableSQLBuilder`. This can be used to generate `table.go` and `view.go` by using `fileTypes` variable that is already there. Thoughts?